website/content/posts/gating-image-promotion-on-gitlab.md

```diff
@@ -0,0 +1,382 @@
+---
+title: "Gating Image Promotion on GitLab"
+date: 2024-06-12T18:64:00-00:00
+author: "Luiz Carvalho"
+---
+
+Once you have a container image ready for promotion, it is important to first verify the image meets
+a certain criteria before it is made available to consumers. In this blog post, we look at how to
```

Comment:
"a certain criteria" sounds a little stiff to me, perhaps something like this:

```suggestion
the standards you've decided to enforce for all promoted images, before it is made available to consumers. In this blog post, we look at how to
```

```suggestion
the build, quality, and security policies you've decided to enforce for all promoted images, before it is made available to consumers. In this blog post, we look at how to
```

Reply:
Thanks, but I think I like it as is better. Let me know if you strongly otherwise.